dependencies_with_url.csv

```diff
@@ -495,3 +495,6 @@ org.fusesource.jansi:jansi:jar:1.16:compile,ASLv2,https://github.com/fusesource/
 com.vividsolutions:jts:jar:1.13:compile
 joda-time:joda-time:jar:2.10:compile
 org.elasticsearch:securesm:jar:1.2:compile
+com.github.stephenc.jcip:jcip-annotations:jar:1.0-1:compile,ASLv2,http://stephenc.github.io/jcip-annotations/
```

Comment:
The "nimbus-jose-jwt" dependency is the JSON web token implementation.  This is used to perform operations on the Knox token that is passed around the various services and UIs.

I believe the "jcip-annotations" is included as a transitive dependency of the "nimbus-jose-jwt".  To be honest I'm not that familiar with JCIP annotations.  Should we try to exclude it?

Reply:
Oh, I know what those are - I thought it sounded familiar, and it's because it's from Brian Goetz's _Java Concurrency In Practice_ - http://jcip.net/. I would imagine the jwt stuff would break at runtime without the JCIP dependency. I'm ok with this dep.